lib/content-services/src/lib/content-node-selector/content-node-selector.component.spec.ts

```diff
@@ -261,6 +261,36 @@ describe('ContentNodeSelectorComponent', () => {
 
     describe('Upload button', () => {
 
+        it('[C588832] Should not be able to upload a file whilst a search is still running', () => {
```

Comment:
with unit tests you don't need a testrail id case, and also you need to move the testrail test C588832  under the folder `DEPRECATED/NOT NEEDED/COVERED BY UNIT TESTS`

Reply:
thanks, i will move it as soon as we merge this PR